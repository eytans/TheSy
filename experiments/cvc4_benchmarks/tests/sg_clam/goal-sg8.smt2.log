19:19:32 [WARN] Collecting statistics
19:19:32 [WARN]  ex is: zero
19:19:32 [WARN]  ex is: (succ zero)
19:19:32 [WARN]  ex is: (succ (succ zero))
19:19:32 [WARN]  ex is: nil
19:19:32 [WARN]  ex is: (cons autovar_Nat_0 nil)
19:19:32 [WARN]  ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:19:32 [WARN] Running exploration without proof mode, but goals were given
19:19:32 [WARN] Proof mode: true
19:19:32 [WARN]  ex is: zero
19:19:32 [WARN]  ex is: (succ zero)
19:19:32 [WARN]  ex is: (succ (succ zero))
19:19:32 [WARN]  ex is: nil
19:19:32 [WARN]  ex is: (cons autovar_Nat_0 nil)
19:19:32 [WARN]  ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:19:32 [WARN] Running exploration without proof mode, but goals were given
19:19:32 [WARN] Proof mode: true
19:19:32 [WARN] Found all lemmas
