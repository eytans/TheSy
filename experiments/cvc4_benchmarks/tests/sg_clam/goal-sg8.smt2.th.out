19:19:32 [0m[33m[WARN] [0mCollecting statistics
(leq ?__x0 ?__y1) => (or (= ?__x0 ?__y1) (less ?__x0 ?__y1))
(or (= ?__x0 ?__y1) (less ?__x0 ?__y1)) => (leq ?__x0 ?__y1)
(drop ?x nil) => nil
(drop zero ?x2) => ?x2
(drop (succ ?x) (cons ?y ?z)) => (drop ?x ?z)
(drop (succ ?w) (drop ?x (cons ?y ?z))) => (drop ?w (drop ?x ?z))
(drop (succ ?v) (drop (succ ?w) (cons ?x (cons ?y ?z)))) => (drop (succ ?v) (drop ?w (cons ?x ?z)))
Searcher: (drop ?autovar14 (cons ?autovar2 ?autovar3))
Searcher: (drop ?autovar14 (drop ?autovar5 (cons ?autovar6 ?autovar7)))
Searcher: (drop ?autovar14 (drop (succ ?autovar9) (cons ?autovar10 (cons ?autovar11 ?autovar12))))
Searcher: (drop (succ ?autovar13) ?autovar15)
19:19:32 [0m[33m[WARN] [0m ex is: zero
19:19:32 [0m[33m[WARN] [0m ex is: (succ zero)
19:19:32 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:19:32 [0m[33m[WARN] [0m ex is: nil
19:19:32 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_0 nil)
19:19:32 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:19:32 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:19:32 [0m[33m[WARN] [0mProof mode: true
19:19:32 [0m[33m[WARN] [0m ex is: zero
19:19:32 [0m[33m[WARN] [0m ex is: (succ zero)
19:19:32 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:19:32 [0m[33m[WARN] [0m ex is: nil
19:19:32 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_0 nil)
19:19:32 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:19:32 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:19:32 [0m[33m[WARN] [0mProof mode: true
19:19:32 [0m[33m[WARN] [0mFound all lemmas
done in 3
